Sources/AppsFlyerRemoteCommand.swift

```diff
@@ -152,84 +152,18 @@ public class AppsFlyerRemoteCommand: RemoteCommand {
     }
     
     func getEventName(command: String) -> String {
-        if let appsFlyerEvent = AppsFlyerConstants.EventCommandNames(rawValue: command.lowercased()) {
-            let eventName = String(standardEventName: appsFlyerEvent)
-            return eventName
-        }
-        return command
+        return AppsFlyerConstants.eventsMap[command.lowercased()] ?? command
     }
 
 }
 
 fileprivate extension Dictionary where Key == String, Value == Any {
     func filterVariables() -> [String: Any] {
-        self.filter {
-            $0.key != "debug" &&
-            $0.key != "method" &&
-            $0.key != "app_dev_key" &&
-            $0.key != "app_id" &&
-            $0.key != AppsFlyerConstants.commandName &&
-            $0.key != "settings"
-        }
+        let excludedKeys: Set<String> = ["method", AppsFlyerConstants.commandName]
+        let allExcludedKeys = excludedKeys.union(AppsFlyerConstants.Configuration.allCases.map { $0.rawValue })
```

Comment:
We can calculate this only once with a `private static let` inside of this extension that we can later reuse in this method.

Something along the lines of:
```swift
private static let allExcludedKeys: Set<String> = {
    let excludedKeys: Set<String> = ["method", AppsFlyerConstants.commandName]
    return excludedKeys.union(AppsFlyerConstants.Configuration.allCases.map { $0.rawValue })
}()
```